### ISSUE TITLE
refreshInterval is being swallowed by clearTimeout

### ISSUE THREAD

**Issue by darrenjennings** [Kong/swrv#101]
refreshInterval seems to be cancelled immediately by https://github.com/Kong/swrv/blob/415f8eb0d6a4dd85cf34c76997308448669c9514/src/use-swrv.ts#L380

Investigate and write tests for refreshInterval working as expected.

**Comment by dcyou** [comment]
hi, I think I have a similar problem, 
maybe due to the duplicate declaration of the timer who erase the other one:
OnMounted: https://github.com/Kong/swrv/blob/next/src/use-swrv.ts#L306
setup: https://github.com/Kong/swrv/blob/next/src/use-swrv.ts#L311

**Comment by darrenjennings** [comment]
@dcyou I have added a test to demonstrate the issue we were running into here:
https://github.com/Kong/swrv/pull/127/files#diff-5da86b7778f76ba7e65c6b2c277c2e5030d665b79a89b8024aa883d182527554R919

Can you let me know if this is the same issue you are seeing?

**Comment by dcyou** [comment]
nice shot! sound like yes based on the test:
works with your change **but** if I remove the change:
![image](https://user-images.githubusercontent.com/6975084/103751521-c16c6900-5008-11eb-90b3-8bc2f0e3971c.png)
so this is like me when the key change the `refreshInterval` is broken
Don't know if it could help my code is similar to:
```
import fetcher from 'lib/API/fetcher'
import useSWRV from 'swrv'
const usePosts = (page, options = {}) => {
    return useSWRV(() => `/user/posts?page=${page.value}`, fetcher, { refreshInterval:2000, ...options })
}
export default usePosts
```

N.B. I will try to implement soon an infinite scroll ;) => https://github.com/Kong/swrv/issues/72

**Comment by darrenjennings** [comment]
great @dcyou, will release soon and look forward to your PR!

**Comment by dcyou** [comment]
ahah thanks not sure when I will be able to do it don't hesitate to do it yourself ^^

**Comment by dcyou** [comment]
I saw you just release a new version for the current version I am using the `next` branch ^^ so 1.0.0-beta6

**Comment by darrenjennings** [comment]
@dcyou thanks for the heads up, I will release one there as well

**Comment by darrenjennings** [comment]
@dcyou released with 1.0.0-beta.7
https://github.com/Kong/swrv/releases/tag/v1.0.0-beta.7
